src/radiant.c

```diff
@@ -1137,6 +1156,19 @@ radiant_dev_t * radiant_open(const char *spi_device, const char * uart_device, i
     dev->thresh[i] = RADIANT_THRESHOLD_UNKNOWN; 
   } 
 
+  if(dev->rad_dateversion_int>230)
```

Comment:
I would set them to 0 unconditionally (but maybe there is already a memset somewhere that does that? I can't remember...)